src/main/java/se/sundsvall/disturbance/integration/db/specification/DisturbanceSpecification.java

```diff
@@ -38,6 +38,10 @@ static Specification<DisturbanceEntity> withPartyId(String partyId) {
 		return (disturbanceEntity, cq, cb) -> cb.like(disturbanceEntity.join(DisturbanceEntity_.AFFECTED_ENTITIES).get(AffectedEntity_.PARTY_ID), partyId);
 	}
 
+	static Specification<DisturbanceEntity> withMunicipalityId(String municipalityId) {
+		return (disturbanceEntity, cq, cb) -> cb.like(disturbanceEntity.get(DisturbanceEntity_.MUNICIPALITY_ID), municipalityId);
+	}
```

Comment:
I think it would be better to use cb.equal() for the municipalityId.

Hudiksvall have municipalityId "22". So 22 would pass as a valid id and this specification could possibly have 9 hits for "22".

Reply:
Yes, I agree. cb.equal is better. Fixed!